package.json

```diff
@@ -93,6 +94,7 @@
     "rimraf": "^6.0.1",
     "tailwindcss": "^3.4.17",
     "typescript": "~5.7.3",
+    "uint8-base64": "^0.1.1",
```

Comment:
You can update to v1.0.0: https://github.com/cheminfo/uint8-base64/releases/tag/v1.0.0